client/app/account/billing/autoRenew/billing-autoRenew.html

```diff
@@ -1,321 +1,322 @@
-<div id="billing-autorenew">
-
-    <div class="page-header">
-        <h1 data-translate="autorenew_title"></h1>
-    </div>
+<div class="page-header">
```

Comment:
Didn't do much there, only added the possibility to have substates and changed the order of elements (for example: alerts are displayed across sub-states)